templates/pages/login.html

```diff
@@ -42,9 +42,15 @@ <h1 class="title">{{ _("mas.login.headline") }}</h1>
 
       <input type="hidden" name="csrf" value="{{ csrf_token }}" />
 
-      {% call(f) field.field(label=_("common.username"), name="username", form_state=form) %}
-        <input {{ field.attributes(f) }} class="cpd-text-control" type="text" autocomplete="username" autocorrect="off" autocapitalize="off" required />
-      {% endcall %}
+      {% if features.login_with_email_allowed %}
+        {% call(f) field.field(label=_("common.username_or_email"), name="username", form_state=form) %}
```

Comment:
The `common.*` and `action.*` namespaces are for strings we share across project, and 'username_or_email' doesn't exist (yet?).

I'd prefer if you used `mas.login.username_or_email` here